client/components/plans/plan-price/index.jsx

```diff
@@ -20,6 +25,11 @@ module.exports = React.createClass( {
 				return this.translate( 'Free', { context: 'Zero cost product price' } );
 			}
 
+			if ( abtest( 'monthlyPlanPricing' ) === 'monthly' && this.props.isInSignup ) {
+				const monthlyPrice = +( rawPrice / 12 ).toFixed( 2 );
+				formattedPrice = formattedPrice.replace( rawPrice, monthlyPrice );
```

Comment:
I agree that it's a bit flaky here - I was just looking for a quick way to substitute the price while keeping the currency in the right place (it might be in front of the price or after, I don't know). AFAIK, we don't offer any upgrades (especially plans) with prices with decimal points, so replacing the `rawPrice` with `monthlyPrice` should always work. The problem here however is that `toFixed` will always use a dot for decimal separator, which might not be true for all locales.